src/ansys/fluent/core/utils/dump_session_data.py

```diff
@@ -121,16 +121,17 @@ def get_surface_data(self, surface_ids, data_type) -> list[Union[np.array, None]
 
         surface_data = []
         for surface_id in surface_ids:
-            surface_data.append(
-                self._session_data["fields"][tag_id][surface_id][
-                    enum_to_field_name[data_type]
-                ]
-            )
+            for data_type in data_types:
+                surface_data.append(
+                    self._session_data["fields"][tag_id][surface_id][
+                        enum_to_field_name[data_type]
+                    ]
+                )
 
         return surface_data
```

Comment:
It's fairly appealing to write this as one line, `return [< list comprehension >]`